addons/account/models/account_move.py

```diff
@@ -46,6 +46,9 @@ def init(self):
             CREATE INDEX IF NOT EXISTS account_move_payment_idx
             ON account_move(journal_id, state, payment_state, move_type, date);
         """)
+        # Create an index for journal dashboard sql query in get_line_graph_datas()
+        self.env.cr.execute('DROP INDEX IF EXISTS account_move_journal_date_desc_idx')
```

Comment:
Why the DROP ?

Reply:
Why the drop? You could just `CREATE INDEX IF NOT EXIST`
Do you have a benchmark or a query plan?
This is probably already included in `account_move_payment_idx` and that index has been improving the performances of the dashboard on multiple databases.
Did you not test it before this was merged? https://github.com/odoo/odoo/pull/82707

Reply:
Of course. Can be changed